src/main/java/com/googleinterns/zoomtube/servlets/LectureServlet.java

```diff
@@ -14,16 +14,16 @@
 
 package com.googleinterns.zoomtube.servlets;
 
-import java.util.*; 
-import java.util.regex.*;
-import com.google.gson.Gson;
-import com.google.sps.data.Lecture;
 import com.google.appengine.api.datastore.DatastoreService;
 import com.google.appengine.api.datastore.DatastoreServiceFactory;
 import com.google.appengine.api.datastore.Entity;
 import com.google.appengine.api.datastore.PreparedQuery;
 import com.google.appengine.api.datastore.Query;
+import com.google.gson.Gson;
+import com.google.sps.data.Lecture;
 import java.io.IOException;
+import java.util.*;
+import java.util.regex.*;
```

Comment:
Wildcard imports are not allowed by the styleguide: https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports

Reply:
fixed